docker/tensorflow-aarch64/README.md

```diff
@@ -15,7 +15,7 @@ aarch64
   * OS: Ubuntu 18.04
   * Compiler: GCC 9.3.0
   * Maths libraries: [Arm Performance Libraries](https://developer.arm.com/tools-and-software/server-and-hpc/compile/arm-compiler-for-linux/arm-performance-libraries) 20.2.1 and [OpenBLAS](https://www.openblas.net/) 0.3.9
-  * [oneDNN](https://github.com/oneapi-src/oneDNN) 0.21.3 or 1.5. Previously known as (MKL-DNN/DNNL).
+  * [oneDNN](https://github.com/oneapi-src/oneDNN) 0.21.3 or 1.7-rc. Previously known as (MKL-DNN/DNNL).
```

Comment:
I think this should be `1.7` rather than `1.7-rc` as v1.7 is released now (and the Dockerfile appears to be pulling the 1.7 tag).

Reply:
Done